main/src/library/Channel/Channel.flix

```diff
@@ -83,6 +83,14 @@ namespace Channel {
 
     type alias LockConditionPair = (Lock, Condition)
 
+    pub def getLock(pair: LockConditionPair): Lock = match pair {
```

Comment:
I think it would more normal to just pattern match on the pair or to use `fst/snd` functions :)